print "test exit error"
exit(1)